helm/botkube/templates/clusterrolebinding.yaml

```
@@ -13,6 +13,24 @@ roleRef:
   kind: ClusterRole
   name: {{ include "botkube.fullname" . }}-clusterrole
 subjects:
+- kind: ServiceAccount
+  name: {{ include "botkube.serviceAccountName" . }}
+  namespace: {{ .Release.Namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: botkube-plugins-read-only
+  labels:
+    app.kubernetes.io/name: {{ include "botkube.name" . }}
+    helm.sh/chart: {{ include "botkube.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: botkube-plugins-read-only
+subjects:
 - kind: ServiceAccount
   name: {{ include "botkube.serviceAccountName" . }}
   namespace: {{ .Release.Namespace }}
```

The subject should be a group we will use for Kubeconfig creation. E.g. as you already defined `botkube-plugins-read-only` in the `context.rbac` group mapping, then:

```suggestion
- kind: Group
  name: botkube-plugins-read-only
  apiGroup: rbac.authorization.k8s.io  
```